images/20-progs/Dockerfile

```diff
@@ -26,3 +26,10 @@ COPY --from=k3os /output/ /output/
 WORKDIR /output
 RUN git clone --branch v0.7.0 https://github.com/ahmetb/kubectx.git \
  && chmod -v +x kubectx/kubectx kubectx/kubens
+# ETCD
+ENV ETCD_VER v3.4.7
+ENV ETCD_DOWNLOAD_URL https://github.com/etcd-io/etcd/releases/download
+RUN apk -U add curl
+RUN curl -L ${ETCD_DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o ${ETCD_VER}-linux-amd64.tar.gz
```

Comment:
How about arm64?

Reply:
@zimme good point, will update.

Reply:
I'm thinking that some docs and ignore scenario should be added when k3os is running on arm7 as etcd don't support arm7 as far as I can tell.

Reply:
According to the K3os docs the os support is ARM64 and x86_64. @dweomer is arm7 a supported arch?

Reply:
Ok I can see that's for a takeover installation, arm7 is still listed, so needs to be handled.

Reply:
PR Updated:

 - Add conditional checks so that etcd binaries are only downloaded for AMD64 and ARM6
 - Improve etcd service running status and quieten messages regarding removing etcd run level
 - Change etcd to be passed by value to avoid nil pointer references if the configuration is missing
 - Add documentation that etcd is available for AMD64 and ARM64

Reply:
PR Updated:
 - Ensure configuration is written to /etc/conf.d/etcd-service and not /etc/conf.d/etcd-service.conf
 - Add new line after network-online in /etc/conf.d/etcd-service